docs/plugins/event_listeners.rst

```diff
@@ -1,2 +1,153 @@
 Event listeners
 ===============
+
+Mautic leverages Symfony's EventDispatcher to execute and communicate various actions through Mautic. Plugins can hook into these to extend the functionality of Mautic. Refer to the Extending Mautic section of the documentation for some of the ways to do this.
+
+.. code-block:: php
+
+    <?php
+    //plugins\HelloWorldBundle\EventListener\LeadSubscriber
+
+    namespace MauticPlugin\HelloWorldBundle\EventListener;
+
+    use Mautic\LeadBundle\Event as Events;
+    use Mautic\LeadBundle\LeadEvents;
+
+    /**
+     * Class LeadSubscriber
+     *
+     * @package Mautic\LeadBundle\EventListener
+     */
+    class LeadSubscriber extends CommonSubscriber
+    {
+    
+        /**
+         * @return array
+         */
+        static public function getSubscribedEvents()
+        {
+            return array(
+                LeadEvents::LEAD_POST_SAVE     => array('onLeadPostSave', 0),
+                LeadEvents::LEAD_POST_DELETE   => array('onLeadDelete', 0)
+            );
+        }
+    
+        public function onLeadPostSave(LeadEvent $event)
+        {
+            $lead = $event->getLead();
+            
+            // do something
+        }
+    
+        public function onLeadDelete(LeadEvent $event)
+        {
+            $lead = $event->getLead();
+            
+            $deletedId = $lead->deletedId;
+            
+            // do something
+        }
+    }
+    // ...
+
+Subscribers
+-----------
+The easiest way to listen to various events is to use an event subscriber. Read more about subscribers in `Symfony's documentation<http://symfony.com/doc/current/components/event_dispatcher/introduction.html#using-event-subscribers>`_. 
```

Comment:
Please use the make link tool at the command line to generate a link file (check there isn't one there already, and please use the similar syntax to the existing link files). Documentation on this feature can be found here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-the-external-links-extension.